app/Http/Requests/Professor/StoreRequest.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace App\Http\Requests\Professor;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class StoreRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        return true;
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'name' => ['required', 'max:100',],
+            'email' => ['required', 'max:50',],
+            'public_email' => ['max:50'],
+            'password' => ['required'],
+            'password-confirmation' => ['required','same:password'],
```

Comment:
Você pode substituir isso adicionando outra regra para `password`: [`confirmed`](https://laravel.com/docs/8.x/validation#rule-confirmed).

Reply:
E como ficaria isso na view? o input password-confirmation deixaria de existir ou precisaria ser modificado de alguma forma?

Reply:
Como tem na documentação: "For example, if the field under validation is `password`, a matching `password_confirmation` field must be present in the input."